controllers/updateservice_controller.go

```diff
@@ -395,6 +428,12 @@ func (r *UpdateServiceReconciler) ensureDeployment(ctx context.Context, reqLogge
 		updated.Spec.Template.ObjectMeta.Annotations[key] = value
 	}
 
+	imageSHA, err := r.ensureGraphDataSHA(ctx, reqLogger, instance, resources)
+	if len(imageSHA)>0 && err == nil{
```

Comment:
how do we want to handle the `err != nil` case?  Can we get that bubbled up to our UpdateService `status` somehow?  Possibly by returning `err` here?

Reply:
fixed